### DISCUSSION TITLE
Help understanding this example: "Transmittance Spectrum of a Waveguide Bend"

### DISCUSSION THREAD

**Discussion by 10UAikawa** [NanoComp/meep#2706] [Q&A]
In this example they calculate the transmission, reflection and loss. It seems the way in which the parameters are calculated is a bit unique to the situation, because the waveguide bends. So, for example, as quoted for the documentation: "in order to normalize the transmitted flux by the incident power to obtain the transmittance, we'll have to do two runs, one with and one without the bend (i.e., a straight waveguide)". 

However, for my case I will have a structure that will have radiation incident on it, and it will transmit through the structure along the z-direction in a straight line. This system is very simple in comparison to a bending waveguide. 

**Questions:**
-So, my question would be: do I still have to run two simulations, since I don't have such a complex structure as in the example? 

-Also why did they run two simulations one with and without the waveguide bend? Why didn't they consider the whole structure to begin with?

-How should I go about doing normalization for my structure? Should I just consider my systems without any objects except the cell and run the first simulation? Then run the second simulation including my object.

**Here is an image of my system for context:**
![image](https://github.com/NanoComp/meep/assets/71509442/17850128-38bd-4a0d-a7f4-273521352571)

**Comment by peterropac** [comment]
-In the case where your structure would not reflect and radiation you could have a monitor before the structure and another one after, and you would use the accumulated fields on the monitor before the structure to normalize the accumulated in the monitor after the structure. Since pretty much every structure will reflect some radiation it's best to use 2 simulations like in the example.

-They use the transmission of the straight to normalize the transmission of the bent waveguide so that they can measure how much of the transmission was lost due to the bend,

-How you normalize you transmission through the structure depends what you are trying to find out. If you are comparing the transmission properties of your structure to a pane of glass, than in your normalization simulation you should replace your structure with a pane of glass. If you are comparing your structure to air/vacuum than do as you suggested.

**Reply by 10UAikawa** [reply]
Thanks for the response! I guess the thing that I am struggling with is not knowing what I should normalize my structure to. From your response if I have a piece of glass that I want to find the transmission through I can normalize that with respect to the air/vacuum. So, in that case should I still normalize with respect to the air/vacuum if my structure is composed of metals in **light blue** and **orange** and a quartz dielectric in **green**. I attached a clearer picture of my structure for context.

Thanks for all of your help!!

**This is a picture of my structure:**
![image](https://github.com/NanoComp/meep/assets/71509442/733b53f8-c581-4133-b5f6-2f803773f447)

**Reply by stevengj** [reply]
If you want transmission compared to the incident power, then you should normalize with a calculation that *only has the incident medium*.

In the case of the waveguide, the incident medium is a straight waveguide.  In your case (with or without a substrate), the incident medium is air/vacuum.
